lib/solr.ts

```diff
@@ -127,6 +127,100 @@ export class Client {
     this.TERMS_HANDLER = 'terms';
   }
 
+  /**
+   * Construct the full path to the given "handler".
+   *
+   * @param handler
+   *   Relative URL path for the solr handler.
+   *
+   * @returns
+   *   Full URL to the handler.
+   */
+  private getFullHandlerPath(handler: string): string {
+    let pathArray;
```

Comment:
path.resolve wouldn't do same thing in a simpler way?

Reply:
That's possible. We can optimize in a next PR.

Reply:
This code was moved / consolidated from e.g. here https://github.com/lbdremy/solr-node-client/pull/280/files/5531fb5405c18ccb0f15284a633474279d710544#diff-3b89c2e1c7e4e714474d6a2f17b55814a790482f4ef375d4f84d4895554c20d9L584 So it's not new. Just refactored.